src/python/pants/option/native_options.py

```diff
@@ -23,7 +23,7 @@
 logger = logging.getLogger()
 
 
-def parse_dest(*args, **kwargs):
+def parse_dest(*args: str, **kwargs) -> str:
```

Comment:
Turns out this annotation means "all *args must be strings" to mypy, which is good.